in_toto/gpg/common.py

```diff
@@ -730,7 +730,13 @@ def parse_signature_packet(data, supported_signature_types=None,
     if subpacket_type == SIG_CREATION_SUBPACKET:
       info["creation_time"] = struct.unpack(">I", subpacket_data)[0]
 
-    info["subpackets"][subpacket_type] = subpacket_data
+    info["subpackets"][subpacket_type] = (subpacket_data, is_hashed)
+
+  # Warn if expiration subpacket is not hashed
+  if KEY_EXPIRATION_SUBPACKET in (info["subpackets"]):
+    if not (info["subpackets"][KEY_EXPIRATION_SUBPACKET][1]):
+      log.warning("Expiration subpacket not hashed, gpg client possibly "
+        "exporting a weakly configured key.")
```

Comment:
Could you please add two more spaces before `"exporting a weakly configured key.")`, our code style guideline prescribes double indentation for continued lines.